airflow/providers/snowflake/CHANGELOG.rst

```diff
@@ -19,6 +19,20 @@
 Changelog
 ---------
 
+2.4.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Support insecure mode in SnowflakeHook (#20106)``
+* ``Remove unused code in SnowflakeHook (#20107)``
+
+.. Below changes are excluded from the changelog. Move them to
+   appropriate section above if needed. Do not delete the lines(!):
+   * ``Fix MyPy Errors for Snowflake provider. (#20212)``
```

Comment:
3 changes that were made to the Snowflake provider are missing but I don't know if this is a problem.
https://github.com/apache/airflow/pull/20245
https://github.com/apache/airflow/pull/20395
https://github.com/apache/airflow/pull/20095

Reply:
Good you noitced. The first 2  are "setup.py" changes and they are not - as such - visible in provider changes. There is currently no link between such "setup.py-only" changes and providers - but we can add them manually. The last one is test-only change and those are not included (and should not be).

Reply:
Updated manually.